frappe_slack_connector/hooks.py

```diff
@@ -152,8 +152,10 @@
 # ---------------
 
 scheduler_events = {
-    "all": ["frappe_slack_connector.tasks.attendance_summary.attendance_channel"],
-    "daily": ["frappe_slack_connector.tasks.send_daily_reminder.send_reminder"],
+    "all": [
+        "frappe_slack_connector.tasks.attendance_summary.attendance_channel",
+        "frappe_slack_connector.tasks.send_daily_reminder.send_reminder",
```

Comment:
Why we do this 🤔 ?

Reply:
Daily is running everyday at 12AM, so as we implemented the scheduler to run at `all` like in attendance_channel, to have control over the time notification is sent

Reply:
As per discussion, we can change it to hourly too